Analysis/SLDCorrection/src/FlightDirection.cc

```diff
@@ -33,6 +33,12 @@ int getRecoFlightDirection( const RecoParticle &linkedRecoLepton , TVector3 &rec
 	daughterHadronFlightDistance = 0.0;
 	daughterHadronFlightDirection = TVector3( 0.0 , 0.0 , 0.0 );
 	sldVertexPosition.clear();
+	if ( linkedRecoLepton->getTracks().size() == 0 )
```

Comment:
```suggestion
	if ( linkedRecoLepton->getTracks().empty() )
```
This is the recommended (and more readable) way for checking whether a vector has no elements.

Reply:
Thanks! Will include this suggestion here and everywhere where the same check occurs.